catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -131,6 +132,9 @@
 	public WebConnectionHolder webConnectionHolder;
 
 	private List<Sprite> sprites;
+	private Sprite spriteToFocusOn = null;
```

Comment:
Please take all changes you made in the StageListener and put it into a seperateClass as the StageListener class already has lots of code in it. You can then create a new member of this class for the StageListener and there call the methods.